packages/styles/src/components/form-check.scss

```diff
@@ -11,6 +11,7 @@
 @use '../mixins/utilities' as utility-mx;
 
 .form-check {
+  align-items: center;
```

Comment:
This won't work if the label text breaks on two lines. After a style update, the checkbox is now 22px high and the label 24px. This means that the checkbox needs a margin-top of 1px to re-center it with the label.

Reply:
I'm not quite sure if i understood the comment correctly. I just added the top-margin to the checkbox/radio and the alignment looks to be correct now.